### PR TITLE
SQL-2387: upload jdbc docs

### PR DESCRIPTION
**Pull request by terakilobyte**
Making docs! https://spruce.mongodb.com/task/mongo_jdbc_driver_eap_release_make_docs_patch_5b24dc527da89971c1d713609a3b8bb12beec1f8_672a538144a808000739cf7f_24_11_05_17_19_09/files?execution=0

### CODE REVIEW

File: evergreen/make_docs.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+DIR="$PWD"
+
+if [[ $(lsb_release -is 2>/dev/null) == "Ubuntu" ]]; then
+    sudo apt install texlive texlive-latex-extra texlive-fonts-recommended -y
+else
+    echo "skipping installation of deps for non-Ubuntu OS"
+fi
+
+MKFILE="$DIR/docs/overview.md"
```
Comment (mattChiaravalloti) [inline evergreen/make_docs.sh:11]: Why "MKFILE" for this name? Sounds like that implies some kind of `make` file

**Review by mattChiaravalloti — APPROVED** [review]
lgtm 👍

**Review by bucaojit — APPROVED** [review]
LGTM

File: evergreen/make_docs.sh (hunk 1 above)
Comment (terakilobyte) [reply evergreen/make_docs.sh:11]: Oh no reason, just a name!

**Pull request merged**